src/uct/ib/rc/base/rc_iface.c

```diff
@@ -78,6 +82,16 @@ ucs_config_field_t uct_rc_iface_config_table[] = {
    ucs_offsetof(uct_rc_iface_config_t, tm.list_size), UCS_CONFIG_TYPE_UINT},
 #endif
 
+#if HAVE_IBV_EXP_DM
+  {"DM_SEG_SIZE", "0",
+   "DM segment size",
```

Comment:
plz add better descriptions. Why is this 0 by default?

Reply:
description updated.
0 - disabled by default